README.md

```diff
@@ -18,5 +18,5 @@ Wonk-mode:
 The txt is actually generated from the xml so it's better if you edit the latter.
 You also get extra brownie points if you:
 * install xml2rfc (`pip install xml2rfc`)
-* generate the txt from the xml
+* generate the txt from the xml (`xml2rfc draft-kelly-json-hal.xml --text`)
```

Comment:
This command didn't work for me. Running `xml2rfc` causing an error.

On my Mac I had to run:
`python /Library/Python/2.7/site-packages/xml2rfc/run.py draft-kelly-json-hal.xml `